### PR TITLE
Refactor/db

### PR DESCRIPTION
**Pull request by Nogesma**

### CODE REVIEW

File: backend/src/models/TFASecret/TFASecret.service.ts [hunk 1]
```diff
@@ -9,11 +10,9 @@ export class TFASecretService {
     private tfaSecretModel: typeof TFASecret
   ) {}
 
-  createTFASecret = (user: number, secret: string, temp: boolean) =>
+  createTFASecret = (user: User, secret: string, temp: boolean) =>
     this.tfaSecretModel.create({ user, secret, temp });
 
-  setTFASecretTemp = (user: number, temp: boolean) =>
+  setTFASecretTemp = (user: User, temp: boolean) =>
```
Comment (Nogesma) [inline backend/src/models/TFASecret/TFASecret.service.ts:16]: We need to revert this change, as tfa secret only has a user id.

**Review by Leo-Gyger — APPROVED** [review]
Thanks for fixing the Database I can once more advance on my branch now
